app/views/hyrax/users/_vitals.html.erb

```diff
@@ -8,7 +8,8 @@
 </div>
 
 <div class="list-group-item">
-  <span class="badge"><%= number_of_works(user) %></span>
+  <%# "Note the `{ generic_type_sim: "Work" }` for number_of_works aligns with the link_to_field's {generic_type: 'Work'} key/value pair." %>
+  <span class="badge"><%= number_of_works(user, where: { generic_type_sim: "Work" }) %></span>
```

Comment:
can we extract this where argument from the view? i'm not a fan of view code knowing about index configuration details.

maybe the `where:` argument could be the default for `number_of_works`?